raknet/src/main/java/com/nukkitx/network/raknet/RakNetSession.java

```diff
@@ -680,20 +681,24 @@ public void send(ByteBuf buf, RakNetPriority priority, RakNetReliability reliabi
     }
 
     public void send(ByteBuf buf, RakNetPriority priority, RakNetReliability reliability, @Nonnegative int orderingChannel) {
+        this.send(buf, priority, reliability, orderingChannel, true);
+    }
+
+    public void send(ByteBuf buf, RakNetPriority priority, RakNetReliability reliability, @Nonnegative int orderingChannel, boolean needsBAS) {
```

Comment:
If `needsBAS` is default for all packets, we don' necessary need this to be parameter here.